test/opt/trim_capabilities_pass_test.cpp

```diff
@@ -3543,6 +3543,69 @@ TEST_F(TrimCapabilitiesPassTest,
   EXPECT_EQ(std::get<1>(result), Pass::Status::SuccessWithoutChange);
 }
 
+TEST_F(TrimCapabilitiesPassTest, QuadControlKHR_RemoveIfNotUsed) {
+  const std::string kTest = R"(
+               OpCapability Shader
+               OpCapability QuadControlKHR
+; CHECK-NOT:   OpCapability QuadControlKHR
+               OpExtension "SPV_KHR_quad_control"
+; CHECK-NOT:   OpExtension "SPV_KHR_quad_control"
```

Comment:
Is this extension on the allow list on the 3 or 4 passes that have them?

Reply:
Not yet, I was thinking of doing that in a separate PR but I suppose doing all together is a bit more convenient.

Reply:
Done.